src/registration/utils.py

```diff
@@ -20,6 +18,12 @@ def get_site(request):
     not installed.
 
     """
+    from django.contrib.sites.models import Site
+    try:
+        from django.contrib.sites.models import RequestSite
+    except ImportError:
+        from django.contrib.sites.requests import RequestSite
```

Comment:
And put it in https://github.com/lambdalisue/django-inspectional-registration/blob/develop/src/registration/compat.py